airbyte-ci/connectors/connector_ops/connector_ops/utils.py

```diff
@@ -645,6 +645,23 @@ def cloud_usage(self) -> Optional[str]:
 
         return get(connector_entry, "generated.metrics.cloud.usage")
 
+    @property
+    def sbom_url(self) -> Optional[str]:
+    """
+    Fetches SBOM URL from the connector definition in the OSS registry, if it exists, None otherwise.
+    """
```

Comment:
```suggestion
        """
        Fetches SBOM URL from the connector definition in the OSS registry, if it exists, None otherwise.
        """
```